ReactSkia/sdk/FollyTimer.cpp

```diff
@@ -28,6 +28,20 @@ Timer::Timer(double duration,
 
 }
 
+Timer::~Timer(){
+  timerCallback_.cb = nullptr;
+
```

Comment:
add comment here
----------
cancelALl need to be called from the same thread of scheduled .